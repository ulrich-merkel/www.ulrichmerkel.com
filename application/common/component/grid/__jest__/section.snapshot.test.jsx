/* eslint-disable import/no-extraneous-dependencies, func-names */
import React from 'react';
import renderer from 'react-test-renderer';
import ComponentToBeTested from './../section';

describe('component/grid/section', function () {
    describe('Snapshot', function () {
        it('should render correctly', function () {
            const tree = renderer.create(
                <ComponentToBeTested>
                    Hello
                </ComponentToBeTested>
            ).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
