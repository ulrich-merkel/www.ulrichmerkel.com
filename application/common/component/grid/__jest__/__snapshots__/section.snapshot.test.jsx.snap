// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/component/grid/section should render correctly 1`] = `
<section
  className="l-grid m-section"
>
  Grid Section Children
</section>
`;
